specification/azureactivedirectory/resource-manager/readme.md

```diff
@@ -24,16 +24,25 @@ For other options on installation see [Installing AutoRest](https://aka.ms/autor
 
 These are the global settings for the azureactivedirectory.
 
-```yaml
+``` yaml
 openapi-type: arm
-tag: package-2017-04-01
+tag: package-2018-01
 ```
 
+
+### Tag: package-2018-01
+
+These settings apply only when `--tag=package-2018-01` is specified on the command line.
+
+```yaml $(tag) == 'package-2018-01'
+input-file:
+  - Microsoft.Aadiam/stable/2018-01-01/azureactivedirectory.json
```

Comment:
You should put the swagger files `metrics_API.json` and `metricDefinitions_API.json` in this section